cmd/mt-whisper-importer-writer/main.go

```diff
@@ -27,7 +29,8 @@ import (
 var (
 	confFile        = flag.String("config", "/etc/metrictank/metrictank.ini", "configuration file path")
 	exitOnError     = flag.Bool("exit-on-error", false, "Exit with a message when there's an error")
-	httpEndpoint    = flag.String("http-endpoint", "127.0.0.1:8080", "The http endpoint to listen on")
+	listenAddress   = flag.String("listen-address", "127.0.0.1", "The address to listen on")
+	listenPort      = flag.Int("listen-port", 8080, "The port to listen on")
```

Comment:
why are you changing this? the commit message "importer transfers cwrs without org" doesn't give any clues..
why not keep this consistent with http.listen? (the config option used for MT to establish its listen address and port)

Reply:
right, i think this temporarily made sense during some older revision, but now not anymore. changing it back